src/Node/DelimitedList.php

```diff
@@ -31,4 +31,17 @@ public function addToken($node) {
         }
         $this->children[] = $node;
     }
+
+    /**
+     * @return Node[]
+     */
+    public function getListElements() {
+        $listElements = [];
+        foreach ($this->children as $child) {
+            if($child instanceof Node) {
+                $listElements[] = $child;
```

Comment:
Would it make sense for DelimitedList to have a $delimiter prop instead, to keep them more flexible? We already have DelimiterLists that do not have a constant delimiter and there may be more.